implementations/index.md

```diff
@@ -57,6 +57,7 @@ isomorphic ActiveRecord clone that issues JSON:API requests instead of SQL and i
 
 ### <a href="#client-libraries-typescript" id="client-libraries-typescript" class="headerlink"></a> Typescript
 * [ts-angular-jsonapi](https://github.com/reyesoft/ts-angular-jsonapi) A JSON:API library developed for AngularJS in Typescript
+* [nestjs-json-api](https://github.com/ringcentral/nestjs-json-api) Module for NestJS and TypeORM. Module to implement JSON API, such as, end point, query params, body params, validation and transformation response.
```

Comment:
Let me propose a small rewording to make it fit better with the other items in the list.

```suggestion
* [nestjs-json-api](https://github.com/ringcentral/nestjs-json-api) A module for NestJS and TypeORM to implement JSON API supporting features such as, end point, query params, body params, validation and transformation response.
```

Reply:
I have added your remarks.